### ISSUE TITLE
Installed partition won't boot

### ISSUE THREAD

**Issue by w0lfschild** [ParasiteTeam/installer#5]
OSX will not finish booting with this installed. Stalls out at the apple progress bar. 

How might I go about debugging this issue?

**Comment by alexzielenski** [comment]
Hey, sorry about this

If you can boot into single user mode (hold CMD+S before apple screen), try deleting `/Library/Extensions/Parasite.kext`
Otherwise, boot into recovery (hold CMD+R), mount your boot disk, open terminal, and delete the same file.

Could tell me what OS version you are running and your method of installing parasite.

**Comment by w0lfschild** [comment]
I was more wondering about how I would go about figuring out the cause.

Would a verbose boot show where it's failing?
Or is there a log of what's happening at some point?

I'll have to try this again on a fresh install some time.

**Comment by shinvou** [comment]
Yeah, a verbose boot would definitely show something. Parasite logs every inject, so that may help.

**Comment by justin0a0** [comment]
You use file vault?

**Comment by w0lfschild** [comment]
Negatory on File Vault. Where are inject logs saved?

**Comment by shinvou** [comment]
Parasite logs to syslog.
